configuration/pih/scripts/global/ces.js

```diff
@@ -304,3 +304,49 @@ function printPrescription() {
       w.close();
   }, 1000);
 }
+
+function setupRespiratoryRisk(cigarrosxDia, anosFumando){
+
+    if(parseInt(cigarrosxDia) !== 0 && parseInt(anosFumando) !== 0){
+        
+        calculateEPOC(cigarrosxDia, anosFumando);
+    }
+
+    jq("#AnosFumando input[type='text']").change(function() {
+
+        let cigarrosxDia = jq("#NumeroCigarros input[type='text']").val();
+        let anosFumando = jq("#AnosFumando input[type='text']").val();
+        calculateEPOC(cigarrosxDia, anosFumando);
+
+    });
+
+    jq("#NumeroCigarros input[type='text']").change(function() {
+
+        let cigarrosxDia = jq("#NumeroCigarros input[type='text']").val();
+        let anosFumando = jq("#AnosFumando input[type='text']").val();
+        calculateEPOC(cigarrosxDia, anosFumando);
+
+    });
+
+}
+
+function calculateEPOC(cigarrosxDia, anosFumando){
+
+      let paquetesPorAno = parseInt((cigarrosxDia * anosFumando)/20);
+
+      if(10>=paquetesPorAno>=0){
```

Comment:
```suggestion
      if(10 >= paquetesPorAno){
```

`x >= y >= z` no funciona